main_board/config/memfault_reboot_reason_user_config.def

```diff
@@ -1,2 +1,3 @@
 // Defines "unexpected" reboots
 MEMFAULT_UNEXPECTED_REBOOT_REASON_DEFINE(BatteryRemoved)
+MEMFAULT_UNEXPECTED_REBOOT_REASON_DEFINE(HeartbeatTimeout)
```

Comment:
Might be nice to clarify *who* the heartbeat is against. The Jetson? The sec mcu? The battery?